README.md

```diff
@@ -54,11 +54,12 @@ Further global configuration options include:
 	maxConcurrentCurls=12 # How many curl checks to run at the same time
 	defaultTimeOut=10 # Max timeout of a check in seconds
 	flapRetry=5 # After how many seconds should we re-check any failed checks? (To prevent flapping)
-	title="Status Dashboard" # Title of the webpage
+	title="Status Dashboard" # Title of the webpage 
+	cgi=false # Enable or disable CGI mode
```

Comment:
This comment does not tell me what cgi mode does. Could you please add a small explanation comment what it is and why you would use it?

Reply:
Added below